tensorbay/geometry/vector.py

```diff
@@ -36,10 +36,16 @@ class Vector(UserSequence[float]):
         y: The y coordinate of the vector.
         z: The z coordinate of the vector.
 
-            .. code:: python
+    Examples:
+        *Initialization Method 1:* Init a Vector2D.
```

Comment:
These two examples do not initialize the same object, so the description title is not appropriate.
Maybe it's ok to just delete the two titles.